### ISSUE TITLE
Add a contributor guide

### ISSUE THREAD

**Issue by chrissainty** [Blazored/Modal#138]
We currently don't have a contributor guide and this makes it hard for people to know how they should go about contributing to the project.

Decide on what the contribution guidelines should be and add a new document to the repo to allow new contributors to understand our process.

**Comment by larsk2009** [comment]
Hi @chrissainty, I made an initial setup of [the contributing guide](https://github.com/Blazored/Modal/blob/contributor-guide/.github/contributing.md). Let me know what you think! It was just something I put together quickly but it might be a good starting point.

Also, do you think we should add a code of conduct? If so, we can just use one of the standard ones provided by github I think.

Let me know!

**Comment by chrissainty** [comment]
@larsk2009 that's a great start! I've made a few changes, if you could take a look and see what you think. 

I remove the point about updating the docs stuff. That isn't being deployed anymore and I think we probably should remove it to avoid confusion. I think we should focus on making the WebAssembly sample a living doc. 

Great idea about the code of conduct. We should definitely add one!

**Comment by larsk2009** [comment]
Yeah I agree removing the separate docs. The rest of the changing seem great to me!

**Comment by chrissainty** [comment]
What needs to happen with this now? Do we just need to link to it from our issue templates and the readme?

**Comment by larsk2009** [comment]
Uhm yeah I think the important thing is the contributing file should be in either the root dir, `.github`  or `docs` for github to automagically use it. Maybe we can merge it first, and see if it shows up when making an issue or PR or if we need to link to it in the PR/issue template.

**Comment by chrissainty** [comment]
Sounds like a plan to me
